explorer/src/lib/stores/appStore.js

```diff
@@ -32,8 +43,17 @@ const setNetwork = (network) =>
     network,
   });
 
+/** @param {boolean} darkMode */
+const setTheme = (darkMode) => {
+  set({
+    ...get(store),
+    darkMode,
+  });
+};
+
```

Comment:
If we don't have a switch, I would propose that we remove this for the time-being.

Reply:
One reason I would not remove this is as a helper for tests, as there is no update method on the store.

Reply:
> One reason I would not remove this is as a helper for tests, as there is no update method on the store.

That is easily fixed by mocking what `matchMedia` returns, but I would keep it nonetheless.